src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const basicDiscount = 20;
+  const extendedDiscount = 50;
+
+  switch (true) {
+    case days < 3:
+      return days * cost;
+    case days >= 3 && days < 7:
+      return days * cost - basicDiscount;
+    case days >= 7:
+      return days * cost - extendedDiscount;
+  }
```

Comment:
It works, but switch/case structure should not be used this way.
Rewrite with if/return.